src/js/utilities/ajax.js

```diff
@@ -229,7 +235,7 @@ Crocodoc.addUtility('ajax', function (framework) {
                 return doXDR(url, method, data, ajaxSuccess, ajaxFail);
             } else {
                 // the browser supports XHR and XHR+CORS, so just do a regular XHR
-                return doXHR(url, method, data, headers, ajaxSuccess, ajaxFail);
+                return doXHR(url, method, data, headers, opt.withCredentials === true, ajaxSuccess, ajaxFail);
```

Comment:
Probably should move this up with the other var declarations at L193 for consistency
withCredentials = !!opt.withCredentials